bionic/cache.py

```diff
@@ -15,7 +15,8 @@
 from pathlib import Path
 from urllib.parse import urlparse
 
-from bionic.exception import UnsupportedSerializedValueError
+from bionic.exception import (
+    UnsupportedSerializedValueError, EntitySerializationError)
```

Comment:
super nit: I don't know if we follow this code style but let's sort the imports by name

Reply:
I haven't been doing that but it sounds reasonable to me.